### ISSUE TITLE
'AttributeError: 'Namespace' object has no attribute 'dev''

### ISSUE THREAD

**Issue by KyleKing** [TeamMsgExtractor/msg-extractor#362]
**Bug Metadata**

* Version of extract_msg: 0.41.0
* Your python version: Python 3.10 and 3.8
* How did you launch extract_msg?
  - [x] My command line or
  - [ ] I used the extract_msg package

**Describe the bug**

It looks like `--dev` wasn't completely removed in 0.41.0 (https://github.com/TeamMsgExtractor/msg-extractor/releases/tag/v0.41.0), there is still a check of `options.dev`, which returns an error

**Traceback**

```sh
Traceback (most recent call last):
  File "../extract-msg/0.41.0/bin/extract_msg", line 8, in <module>
    sys.exit(main())
  File "../extract-msg/0.41.0/venv/lib/python3.10/site-packages/extract_msg/__main__.py", line 17, in main
    args = utils.getCommandArgs(sys.argv[1:])
  File "../extract-msg/0.41.0/venv/lib/python3.10/site-packages/extract_msg/utils.py", line 390, in getCommandArgs
    if options.dev or options.fileLogging:
AttributeError: 'Namespace' object has no attribute 'dev'
```

From:

https://github.com/TeamMsgExtractor/msg-extractor/blob/1837ce2a7cc7f692c5f8f90ba48cee9b9ae0716d/extract_msg/utils.py#L390-L391

**Comment by TheElementalOfDestruction** [comment]
Uggg, I knew something would go wrong with removing that option. Thanks for catching this, I'll have a patch out soon

**Comment by TheElementalOfDestruction** [comment]
There w go, fixed and uploaded 0.41.1 to PyPI. Let me know if you have any more issues

**Comment by KyleKing** [comment]
Thank you for the fast fix!
